clients/fides-js/src/lib/i18n/i18n-context.tsx

```diff
@@ -28,17 +27,8 @@ interface I18nProviderProps {
   children: ReactNode;
 }
 export const I18nProvider = ({ i18nInstance, children }: I18nProviderProps) => {
-  const [currentLocale, setCurrentLocale] = useState<string>();
   const [isLoading, setIsLoading] = useState<boolean>(false);
 
-  const onSetCurrentLocale = useCallback(
-    (locale: string) => {
-      i18nInstance.activate(locale);
-      setCurrentLocale(locale);
-    },
-    [i18nInstance, setCurrentLocale],
-  );
-
```

Comment:
This was all overkill and was causing some problems. Just use the ol' `i18n.locale` and `i18n.activate` as intended